src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -147,8 +147,10 @@ public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasE
 
         if (Objects.nonNull(entityDto)) {
             var entityEmail = entityDto.getEmail();
-            // Temporary as initial public data Entity fetch has no Email Address, Needs to change once private data fetch is implemented
-            if (StringUtils.isNotBlank(entityEmail)) {
+            var incorporatingCountry = entityDto.getIncorporationCountry();
```

Comment:
We don't call this function? It's commented out on line 136

Reply:
reverted that line 136